tests/test_log.py

```diff
@@ -34,6 +35,26 @@ def test_logging_consumer():
     kc.close()
 
 
+def test_logging_avro_consumer():
+    """ Tests that logging works """
+
+    logger = logging.getLogger('avroconsumer')
+    logger.setLevel(logging.DEBUG)
+    f = CountingFilter('avroconsumer')
+    logger.addFilter(f)
+
+    kc = confluent_kafka.avro.AvroConsumer({'schema.registry.url': 'http://example.com',
+                                            'group.id': 'test',
+                                            'debug': 'all'},
+                                           logger=logger)
+    while f.cnt == 0:
+        kc.poll(timeout=0.5)
+
+    print('%s: %d log messages seen' % (f.name, f.cnt))
```

Comment:
should add a check that the logger actually saw some logs

Reply:
isn't that implied by the `f.cnt > 0` after the `while` loop?

Reply:
Ah, you're right!